packages/react-storage/src/components/StorageManager/hooks/useStorageManager/useStorageManager.tsx

```diff
@@ -36,14 +36,16 @@ export function useStorageManager(
       action: Action
     ) => UseStorageManagerState
   >(storageManagerStateReducer, {
-    files: defaultFiles.map((file) => {
-      return {
-        ...file,
-        id: file.key,
-        key: file.key,
-        status: FileStatus.UPLOADED,
-      };
-    }) as StorageFiles,
+    files: (defaultFiles ?? [])
```

Comment:
Wherever we put it, we need to nullishly coalesce it. Passing null in as `defaultFiles` argument will override the default array. The `useStorageManager` has a default value for defaultFiles as `[]` already.

Reply:
In typescript we don't, but JS..

Reply:
```
function StorageManager({
  acceptedFileTypes,
  accessLevel,
  defaultFiles = []
```
I've done something like this before and it would lead to a loop that break jest. Don't know if it's relevant here though since `defaultFiles` is probably not used to update a state within `StorageManager`.